src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+      src="/styles/images/imac.jpeg"
+      class="product-card__image"
+      alt ="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+    >
+    </img>
+      <article class="product-card__description">
+        <p class="product-card__item-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="product-card__code">
+          Product code: 195434
+        </p>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__teg">
+            Price:
+          </p>
+
+          <p class="product-card__value">
+            $2,199
+          </p>
+        </div>
+
+        <form
+          action="/buy"
+          method="post"
+          data-qa="hover"
+        >
+          <button type="submit" class="product-card__buy-button">
+            <p>Buy</p>
```

Comment:
"Item with the text `Buy` should be a link" - please, check the Tips & Hints in readme.md file again
So, you should type something like this: `<a data-qa="hover"> Buy </a>`